### PR TITLE
Adds keys and verb for changing limb target zones

### PR DESCRIPTION
**Pull request by ghost**
[content][qol]
Uses numpad keys, be sure to turn on/off numlock so there's no overlaps.
HUJK cycles them up/down/left/right on hotkeys mode.
Will be helpful for macros too.
:cl:
 * rscadd: Targeted zone can now be changed with numpad keys, or HUJK in hotkeys mode

### CODE REVIEW

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -463,6 +463,23 @@ var/list/intents = list(I_HELP,I_DISARM,I_GRAB,I_HURT)
...
 				hud_used.action_intent.icon_state = "help"
 
+//change a mob's zone_sel. Input the target as a string such as LIMB_HEAD (mouth and eyes do not follow this)
+/mob/verb/a_zone_change(input as text)
+	set name = "a-zone"
+	set hidden = 1
+	
+	if(zone_sel && zone_sel.selecting)
+		var/old_selecting = zone_sel.selecting
+		switch(input)
```
Comment (ShiftyRail) [inline code/modules/mob/mob_helpers.dm:491]:
```dm
if (input in list(LIMB_RIGHT_FOOT,LIMB_LEFT_FOOT,LIMB_RIGHT_HAND,LIMB_LEFT_HAND,LIMB_RIGHT_ARM,LIMB_LEFT_ARM,LIMB_RIGHT_LEG,LIMB_LEFT_LEG,LIMB_GROIN,LIMB_CHEST,LIMB_HEAD, "mouth", "eyes")
     zone_sel.selecting = input
```
that would do the same thing. I don't get the switch.
You could go the extra mile and make a define for mouth and eyes if there isn't one already.

Comment (ghost) [reply code/modules/mob/mob_helpers.dm:491]: Done.
There was already a define for mouth that matched, but not eyes, so I added that.

**Comment by Kurfursten** [comment]
code/modules/mob/mob_helpers.dm, line 438, column 19:
warning: proc call requires static type: "Find"
- 422:16: add a type annotation after /list here

**Comment by ghost** [comment]
> code/modules/mob/mob_helpers.dm, line 438, column 19:
> warning: proc call requires static type: "Find"
> 
> * 422:16: add a type annotation after /list here

Done

**Pull request merged**